### PR TITLE
Ports blind people can feel to examine objects

### PR DESCRIPTION
**Pull request by maxion12345**
issa port https://github.com/BeeStation/BeeStation-Hornet/pull/4443
#### Read our contributing.md if you haven't already: https://github.com/yogstation13/Yogstation/blob/master/.github/CONTRIBUTING.md

# General Documentation

### Intent of your Pull Request

Blind people can use empty hands to feel nearby objects to examine them

### Why is this change good for the game?
Lets blind people examine things or people, as long as they have an empty hand
# Wiki Documentation
don't think we have blind people on wiki
### Briefly describe your PR and the impacts of it, in layman's terms. 
Using an empty hand, blind people can examine nearby objects with a delay

### What should players be aware of when it comes to the changes your PR is implementing?
see above
### What general grouping does this PR fall under? 
addition

### Are there any aspects of the PR that you would like us not to mention on the Wiki?
n/a
### If there are any numerical values involved in your PR that will be relevant to a player, please note them here. 
For example, "This new chem will heal X brute damage". 

# Changelog

:cl:  ivanmixo
rscadd: Blind people can now examine things nearby by feeling them up with an empty hand.
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
link the prs your porting

**Review by JamieD1 — APPROVED** [review]
